clang/lib/Lex/PPDirectives.cpp

```diff
@@ -2086,9 +2086,7 @@ Preprocessor::ImportAction Preprocessor::HandleHeaderIncludeOrImport(
   // Issue a diagnostic if the name of the file on disk has a different case
   // than the one we're about to open.
   const bool CheckIncludePathPortability =
-//      !IsMapped && !File->getFileEntry().tryGetRealPathName().empty();
-// Disable file portability check to unblock pulldown
-      false;
+      !IsMapped && !File->getFileEntry().tryGetRealPathName().empty();
```

Comment:
What was the reason to disable it?

Reply:
We had LIT failures on windows after LLVM pulldown (https://github.com/intel/llvm/issues/1594)

Reply:
Doesn't seem that it can be enabled already http://ci.llvm.intel.com:8010/#/builders/18/builds/2264 .